apps/golden-sample-app/src/app/guards/auth.guard.ts

```diff
@@ -1,16 +1,19 @@
-import { Injectable } from '@angular/core';
+import { Inject, Injectable, LOCALE_ID } from '@angular/core';
 import { CanActivate } from '@angular/router';
 import { OAuthService } from 'angular-oauth2-oidc';
 
 @Injectable()
 export class AuthGuard implements CanActivate {
-  constructor(private oAuthService: OAuthService) {}
+  constructor(
+    private oAuthService: OAuthService,
+    @Inject(LOCALE_ID) private locale: string
+  ) {}
 
   canActivate(): boolean {
     if (this.oAuthService.hasValidAccessToken()) {
       return true;
     }
-    this.oAuthService.initLoginFlow();
+    this.oAuthService.initLoginFlow(undefined, { ui_locales: this.locale });
```

Comment:
Can we move this change to a different PR? I'm interested in reviewiing it with the Identity team

Reply:
Done